content/docs/command-reference/move.md

```diff
@@ -161,3 +165,57 @@ $ tree
     │   └── foo
     └── data3.dvc
 ```
+
+## Example: manually renaming stage outputs
+
+`dvc move` doesn't support renaming of <abbr>outputs</abbr> in a stage, you will
+have to do that manually. Let's look at our sample workspace and `dvc.yaml`:
+
+```dvc
+$ tree
+.
+├── dvc.lock
+├── dvc.yaml
+├── keras.h5
+├── train.py
+└── ...
+
+$ cat dvc.yaml
+stages:
+  train:
+    cmd: python train.py
+    deps:
+    - data.csv
+    - train.py
+    params:
+    - epochs
+    - lr
+    outs:
+    - keras.h5
+```
+
+In this example we have to rename `keras.h5` as `model.h5`. First change the
+name of model file in code (`train.py`). Next change output name in `train`
+stage of `dvc.yaml`. After this change `/keras.h5` to `/model.h5` in
+`.gitignore` file. Lastly, rename the existing model file:
+
+```dvc
+$ mv keras.h5 model.h5
+```
+
+In final step run `dvc commit` to save the changes and update the `dvc.lock`
```

Comment:
in final step - grammar

Reply:
the dvc.lock file -> just "dvc.lock" should be enough.

Reply:
Although not sure we even want to mention it, as it's not mentioned much less explained anywhere in the doc. Is there a better phrase that explains what `commit` is for here without that technical detail?

Reply:
May also want to mention why -f

Reply:
changed this to `dvc commit with -f option to force save the changes`. I think this explains minimally without technical details.

Reply:
but what does commit do? Why is force needed?